mobanfile

```diff
@@ -1,8 +1,10 @@
-requires:
-   - pypi-mobans-pkg
+#requires:
+#  - type: git
+#    url: https://github.com/jayvdb/pypi-mobans
+#    branch: setup-env-markers
 configuration:
   template_dir:
-    - "setupmobans:templates"
+    - "../pypi-mobans/templates"
```

Comment:
this bit is dev version. 


and I have not looked at this local repo yet.

Reply:
I do share your feeling as I need to use local git repo too

Reply:
We have an enhancement request about this.

Once the pypi-mobans fix is merged, I will update this.